tests/test_arg_parse.py

```diff
@@ -53,5 +56,6 @@ def test_minimal():
         verbose=False,
         version=False,
         include_prereleases=False,
+        dry_run=False,
```

Comment:
Isn't this the default anyway?

Reply:
Tests fail otherwise here

Reply:
Should I leave it like this or what changes should be made?